test/shared/fixtures.ts

```diff
@@ -238,7 +238,9 @@ export const uniswapFixture: Fixture<UniswapFixtureType> = async (
   )
   const staker = (await stakerFactory.deploy(
     factory.address,
-    nft.address
+    nft.address,
+    2 ** 32,
```

Comment:
can do `constants.MaxUint256` to get old behavior

Reply:
but I want to test it